homeassistant/components/lightwave/__init__.py

```diff
@@ -34,10 +48,19 @@
 
 async def async_setup(hass, config):
     """Try to start embedded Lightwave broker."""
-
     host = config[DOMAIN][CONF_HOST]
     hass.data[LIGHTWAVE_LINK] = LWLink(host)
 
+    trv_proxy_ip = "127.0.0.1"
+    if PROXY_IP in config[DOMAIN].keys():
+        trv_proxy_ip = config[DOMAIN][PROXY_IP]
+    trv_proxy_port = 7878
+    if PROXY_PORT in config[DOMAIN].keys():
+        trv_proxy_port = int(config[DOMAIN][PROXY_PORT])
```

Comment:
Use default values in the config schema instead. See [default values in voluptuous](http://alecthomas.github.io/voluptuous/docs/_build/html/voluptuous.html?highlight=default%20value#voluptuous.schema_builder.Optional).